src/test/prob/normal/normal_sufficient_test.hpp

```diff
@@ -0,0 +1,117 @@
+// Arguments: Doubles, Doubles, Ints, Doubles, Doubles

```

Comment:
Small comment: Shouldn't this test be in src/test/prob/normal_sufficient/ rather than in src/test/prob/normal/?